docs/csharp/language-reference/operators/snippets/SwitchExpressions.cs

```diff
@@ -0,0 +1,137 @@
+using System;
```

Comment:
This file is supposed to be in a sub folder named after the article. (well all of these snippet files should be)

Reply:
@Thraka I like that this one sample program is used for all the articles in this folder. 

I'll switch the article name from *switch.md* to *switch-expression.md*.  

I will be updating the samples based on the other feedback this morning. (My goal is to make everything interactive). 

Once I do that, let's resolve this. I'd like to keep these snippets as one project for simplicity. Beyond that, I'm open to moving things around. Although, for scope, I might push for a 2nd PR to move samples.